python_ta/__init__.py

```diff
@@ -272,21 +273,28 @@ def reset_linter(config=None, file_linted=None):
         ),
     )
 
+    absolute_checkers_path = Path("python_ta//checkers").absolute()
```

Comment:
Using `python_ta/checkers` still produces a path that's relative to where the user is running PythonTA, so that's not what we want. You should use the special `__file__` variable to get the location of this file, and then go from there.